#Make CMake install all binaries
#10th October 2016 by B. Hajgato (Free University Brussels - VUB)
--- bert-master-761b5f7fd1d3d8b27dcded6d0433f7b4de951d22//dcfemlib/apps/CMakeLists.txt.orig	2016-10-07 14:14:36.000000000 +0200
+++ bert-master-761b5f7fd1d3d8b27dcded6d0433f7b4de951d22/dcfemlib/apps/CMakeLists.txt	2016-10-10 16:26:18.364030617 +0200
@@ -15,7 +15,9 @@
 
     target_include_directories(${program} PRIVATE ${DCFEMLIB_INCLUDE_DIR})
 
-    install(TARGETS ${program} DESTINATION bin)
+    install(TARGETS ${program} 
+            RUNTIME DESTINATION bin
+            LIBRARY DESTINATION lib)
     add_dependencies(bert1 ${program})
 endforeach(programCPP)
 
